app/src/main/java/org/jellyfin/androidtv/ui/card/LegacyImageCardView.java

```diff
@@ -22,13 +22,13 @@
  * Modified ImageCard with no fade on the badge
  * A card view with an {@link ImageView} as its main region.
  */
-public class MyImageCardView extends BaseCardView {
-    private ImageCardViewBinding binding = ImageCardViewBinding.inflate(LayoutInflater.from(getContext()), this);
+public class LegacyImageCardView extends BaseCardView {
```

Comment:
Should we be concerned about the missing constructor warning here?

Reply:
For new views I'd complain about this warning, but it's not something that breaks the app. The missing constructor just causes the view to not work in the Android Studio previews. But AFAIK none of the activities/fragments using this card currently work as previews in AS.

Reply:
A proper fix is to remove the `showInfo` parameter in the constructor and make it a class-property.